gips/utils.py

```diff
@@ -269,22 +269,30 @@ def get_data_variables():
     # ex asset = modis
     for driver in sources.keys():
         data_class = import_data_class(driver)
+        print(data_class)
```

Comment:
optional:  In general we're trying to push gips towards acting like a library by default, so we try to avoid print() for new code.